base/range.jl

```diff
@@ -76,6 +76,34 @@ julia> range(1, step=5, stop=100)
 range(start; length::Union{Integer,Nothing}=nothing, stop=nothing, step=nothing) =
```

Comment:
Maybe remove the docstring from this method? Or simplify it significantly? The two are essentially identical and will just end up looking repetitive in the docs.